README.md

```diff
@@ -61,15 +67,32 @@ Then execute the command as normal:
 icgc-get download FI378424 -r  collaboratory
 ```
 
+## Status
+Another useful subcommand is `status`.  This takes the same primary inputs as `download`,
+but instead of downloading the specified files, it will provide a list of all files that are
+about to be downloaded, including their size, data type, and the repository they are hosted on.
+It will also provide a summary of the download by repository and data type, showing how many files
+and the total size of the files for each catagory.  In case of very large downloads, the individual
+file summary may be too large to be practical, and it can be supressed with the flag `-nf`. 
```

Comment:
Please mention the access checking, and for which repos this is done and not done.